packages/frontend/src/components/HourCommentBox.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="comment-container">
+    <div class="comment-box">
+      <label for="timeEntrieComment">Kommentar</label>
+      <textarea
+        id="timeEntrieComment"
+        v-model="comment"
+        :disabled="disabled"
+        @input="onChange"
+        @keydown.tab="onClickOutside"
+      ></textarea>
+    </div>
+  </div>
+</template>
+
+<script>
```

Comment:
Even though AlvTime has quite weak typing, we could convert this to typescript 
Replace 
```html
<script>
```
 with 
```html
<script lang="ts">
```